torch/csrc/jit/mobile/runtime_bytecode_version.cpp

```diff
@@ -0,0 +1,14 @@
+#include <caffe2/serialize/inline_container.h>
+#include <torch/csrc/jit/mobile/runtime_bytecode_version.h>
+
+namespace torch {
+namespace jit {
+namespace mobile {
```

Comment:
Why does this have to live (i.e. be specific) to mobile? I understand bytecode is used for mobile, but kProducedBytecodeVersion is not mobile specific.

@iseeyuan

Reply:
Good point. kProducedBytecodeVersion is not in mobile. The function name _get_runtime_bytecode_version is quite self explainable. kProducedBytecodeVersion is in caffe2::serialize where "caff2" is a legacy name. It's fine to me to just use torch::jit.

Reply:
Resolved in the latest commit

Reply:
Also, does the code need to live in torch/csrc/jit/mobile/ as opposed to torch/csrc/jit/ ?